### PR TITLE
Add akamai fast purge for config files.

### PR DESCRIPTION
**Pull request by Hyperkid123**
jira: https://issues.redhat.com/browse/RHCLOUD-16700

### Changes
- add cache purge to Jenkins 

cc @john-dupuy

### CODE REVIEW

**Review by karelhala — APPROVED** [review]
Codewise looks good!

File: Jenkinsfile [hunk 1]
```diff
@@ -72,6 +72,17 @@ node {
           rsync -arv -e \"ssh -2\" ./chrome/*.json sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}/chrome
         """
       }
+
+      // set akamai fast purge env
+      openShiftUtils.withJnlpNode(image: "quay.io/redhatqe/jenkins-slave-python36-akamai:v3.11") {
+        sh "wget https://raw.githubusercontent.com/RedHatInsights/cloud-services-config/ci-beta/akamai/cache_buster/bust_cache.py"
```
Comment (john-dupuy) [inline Jenkinsfile]: Should we make this `env` aware? E.g. pulling from prod-beta/prod-stable instead of always ci-beta.

File: Jenkinsfile [hunk 2]
```diff
@@ -72,6 +72,17 @@ node {
...
+        // get akamai fast purge credentials
+        withCredentials([file(credentialsId: "jenkins-eccu-cache-purge", variable: 'EDGERC')]) {
+          sh "python3 bust_cache.py $EDGERC ${BRANCH} ${NAVLIST}"
```
Comment (john-dupuy) [inline Jenkinsfile:79]: If running on `stage-stable` or `stage-beta` - `NAVLIST` will be undefined.

File: Jenkinsfile [hunk 3]
```diff
@@ -72,6 +72,17 @@ node {
           rsync -arv -e \"ssh -2\" ./chrome/*.json sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}/chrome
         """
       }
+
+      // set akamai fast purge env
+      openShiftUtils.withJnlpNode(image: "quay.io/redhatqe/jenkins-slave-python36-akamai:v3.11") {
```
Comment (john-dupuy) [inline Jenkinsfile:75]: So that it's clear in the pipeline stage view, I think it would be good to put this in a separate stage (if possible).

**Review by john-dupuy — COMMENTED** [review]
Few comments, I tested running this Jenkinsfile (https://github.com/john-dupuy/cloud-services-config/blob/jdupuy-test-akamai-cache/Jenkinsfile) after fixing the `NAVLIST` and it ran fine. 

EDIT: Just thought of something - if the smoke tests fail and we revert the files, we need to purge the cache there too.

File: Jenkinsfile (hunk 1 above)
Comment (Hyperkid123) [reply Jenkinsfile]: We could. But I don't know if there will be any differences really.

**Review by john-dupuy — APPROVED** [review]
LGTM, thanks for this one @Hyperkid123!

**Comment by Hyperkid123** [comment]
OK lets try this

**Pull request merged**